### PR TITLE
Add EigenDA mainnet parameters

### PR DESCRIPTION
**Pull request by ian-shim**
- updates testnet batch interval 
- adds mainnet parameters

### CODE REVIEW

File: docs/eigenda/networks/mainnet.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ sidebar_position: 3
 
 # Mainnet
 
-The EigenDA mainnet is coming soon. Stay tuned [on twitter][1] for updates.
+## Quick Links
 
-[1]: https://twitter.com/eigen_da
+* [AVS Page][2]
+* [Blob Explorer][1]
```
Comment (jianoaix) [inline docs/eigenda/networks/mainnet.md:10]: Maybe add Churner (Dataapi may be left out at the current stage)

File: docs/eigenda/networks/mainnet.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ sidebar_position: 3
...
+* [Deployed Contract Addresses][3]
+
+## Specs
+
+| Property | Value |
+| --- | --- |
+| Disperser Address | `disperser.eigenda.xyz:443` |
+| Batch Confirmation Interval | Every 10 minutes (may vary based on network health) |
+| Max Blob Size | 2 MB |
+
```
Comment (jianoaix) [inline docs/eigenda/networks/mainnet.md:20]: what about the rate limiting params?

File: docs/eigenda/networks/mainnet.md (hunk 1 above)
Comment (ian-shim) [reply docs/eigenda/networks/mainnet.md:10]: good call. Added churner address below the disperser address

File: docs/eigenda/networks/mainnet.md (hunk 2 above)
Comment (ian-shim) [reply docs/eigenda/networks/mainnet.md:21]: There is no default rate limits on mainnet (no default free tier)

**Review by jianoaix — APPROVED** [review]

**Comment by ian-shim** [comment]
I'll merge this when 1) rate limiter issue is resolved and 2) the public endpoint is open

**Pull request merged**